docs/tasks/debug-application-cluster/events-stackdriver.md

```diff
@@ -0,0 +1,86 @@
+---
+assignees:
+- crassirostris
+- piosz
+title: Events Using Stackdriver
```

Comment:
I wanted it to be consistent with the logging page, but I agree, "Events in Stackdriver" sounds better